mlir/test/Dialect/LLVMIR/debuginfo.mlir

```diff
@@ -155,17 +155,17 @@ llvm.func @addr(%arg: i64) {
   %allocCount = llvm.mlir.constant(1 : i32) : i32
   %alloc = llvm.alloca %allocCount x i64 : (i32) -> !llvm.ptr
 
-  // CHECK: llvm.intr.dbg.declare #[[VAR0]] = %[[ALLOC]]
-  llvm.intr.dbg.declare #var0 = %alloc : !llvm.ptr
+  // CHECK: llvm.intr.dbg.value #[[VAR0]] #llvm.di_expr<[]> = %[[ALLOC]]
+  llvm.intr.dbg.value #var0 #llvm.di_expr<[]> = %alloc : !llvm.ptr
   llvm.return
 }
 
 // CHECK: llvm.func @value(%[[ARG1:.*]]: i32, %[[ARG2:.*]]: i32)
 llvm.func @value(%arg1: i32, %arg2: i32) {
-  // CHECK: llvm.intr.dbg.value #[[VAR1]] = %[[ARG1]]
-  llvm.intr.dbg.value #var1 = %arg1 : i32
-  // CHECK: llvm.intr.dbg.value #[[VAR2]] = %[[ARG2]]
-  llvm.intr.dbg.value #var2 = %arg2 : i32
+  // CHECK: llvm.intr.dbg.value #[[VAR1]] #llvm.di_expr<[2, 4096, 0, 4]> = %[[ARG1]]
+  llvm.intr.dbg.value #var1 #llvm.di_expr<[2, 4096, 0, 4]> = %arg1 : i32
```

Comment:
question: is there any chance we can try to sugar the enums a bit?

Reply:
You mean printing the operator names? I added that as a TODO, mainly because it would involve refactoring some utils from llvm so that we can also understand the operators and how many arguments each has. I was planning on doing that as a followup if you don't have a strong preference.

Reply:
follow up is fine